workers/shared-worker-options-mismatch.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<title>SharedWorker: type or credentials mismatch failure</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script> setup({allow_uncaught_exception: true}); </script>
```

Comment:
Why is this needed?

Reply:
Sorry, this is a mistake. I will remove this line.